lib/openai/assistant_files.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module OpenAI
+  class AssistantFiles
+    class InvalidFileFormat < OpenAI::Error; end
+
+    VALID_EXTENSIONS = %w[
```

Comment:
I think validating at the client level here is a going to be game of whack-a-mole and we should let the server enforce this instead.

Reply:
Agreed. Easier to just allow anything I think?

Reply:
ya, +1

Reply:
Removed 😄